EIPS/eip-7742.md

```diff
@@ -44,37 +44,57 @@ and deployment of changes to the blob count easier.
-with each provided payload over the Engine API. The EL block header will also need to be extended with this target value
-to preserve the security of optimistic sync.
+#### Target blobs per block
+
+The target is currently specified as a fixed value in relation to the blob count. The Ethereum community intends to increase the blob parameters as part of its scaling strategy and the ability to have a more flexible target value in relation to the blob max is desirable to reduce rigidity in this protocol parameter.
+
+To this end CL now provides a blob count target as well during block production and the EL block header is extended with this `target_blob_count` value to preserve the security of optimistic sync as well as for the purpose of `excess_blob_gas` and blob base fee calculations.
+
+#### Updated Fee calculations
+
+With the changing target, we also need to scale the `BLOB_BASE_FEE_UPDATE_FRACTION` from [EIP-4844](./eip-4844.md) accordingly to bound the price jumps by +-12.5%. But this introduces some irregularity on the fork block because excess blob gas is effectively scaled down by this changed factor in the fee calculations.
+
+However we consider the target change events to be few and far between and hence not a cause of concern. The likely effect of this factor update will be to bring down the base fee on the fork block and hence would not negatively impact the transactions in txpool.
 
 ## Specification
 
-### Block structure and validity
+### Execution Block structure and validity
 
 Upon activation of this EIP, execution clients **MUST** extend the header schema with an
-additional 64-bit field: the `target_blob_count`. This value is set to the current target blob count. The Engine API
-is modified along with this EIP to provide the `target_blob_count` with each payload and implementations can use this
-value to correctly set the block header field.
+additional 64-bit field: the `target_blob_count`. This values is set to the one provided by the CL with the modified engine api fcU's payload attributes.
+
+Any reference to `TARGET_BLOB_GAS_PER_BLOCK` from [EIP-4844](./eip-4844.md) (for e.g. for `excess_blob_gas` calculations) can be derived by taking the `target_blob_count` from the EL block header and multiplying by `GAS_PER_BLOB`.
+
+Validity of these values is guaranteed from the consensus layer, much like how withdrawals are handled. Hence when verifying a block, execution clients **MUST** ensure the target blob count in the block header matches the one provided by the consensus client.
+
+For a genesis block with no existing parent, the value should be set according to the agreed specification for the target blob count given by that genesis block's protocol rule set.
+
+Furthermore `get_base_fee_per_blob_gas` as specified by [EIP-4844](./eip-4844.md) is as modified below:
+
+| Constant | Value |
+| - | - |
+| `BLOB_BASE_FEE_UPDATE_FRACTION_PER_TARGET_BLOB` | `834619` |
```

Comment:
where did this number come from?

Reply:
I think for the way you want to use this value (just multiplying some number by the `target_count`) we want a value of `1_112_825`

Reply:
ahh yes, i divied up the original mistakenly by 4